### DISCUSSION TITLE
Should zls be installed with zig?

### DISCUSSION THREAD

**Discussion by hverlin** [jdx/mise#3390] [Q&A] [answered]
I am not sure if [zls](https://github.com/zigtools/zls) should just be installed automatically with `zig` core or simply added to the registry?
 
 For now, available, it's available with `aqua` and `ubi`
```sh
mise use -g aqua:zigtools/zls
mise use -g ubi:zigtools/zls
```

**Comment by jdx** [comment, marked as answer]
we don't include language servers with other tools unless they're bundled with the language so no. We can add it to the registry though.
